_posts/static/2018-02-05-how-to-use.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+howtos: true
+published: true
+title: How to use
+permalink: howtos/how-to-use
+description: Demo plugin + Dashboard
+---
+
+## Demo plugin + Web Dashboard
+
+There are some configurations required in order to use the Demo mode plugin of Flyve MDM with the Web Admin Dashboard.
+
+### User flyvenologin
+
+This user is created automatically once the demo plugin is installed.
+
+Ensure it is activated.
```

Comment:
Flyvenologin isconfigured and enabled or disabled by the configuration page of the plugin. In normal operations, there is no need to edit it manually.

Just document the plugin based on the configuration page.